CHANGELOG.md

```diff
@@ -1,20 +1,20 @@
-## Unreleased
+## v0.42.0 (January 31, 2023)
 
-BUG FIXES:
-* `r/tfe_workspace`: Return all workspace safe deletion errors by @skeggse ([#758](https://github.com/hashicorp/terraform-provider-tfe/pull/758))
+FEATURES:
+* **New Provider Config**: `organization` (or the `TFE_ORGANIZATION` environment variable) defines a default organization for all resources, making all resource-specific organization arguments optional. ([#762](https://github.com/hashicorp/terraform-provider-tfe/pull/762))
```

Comment:
I also DMd you about this - I like it when we list the person whose PR is included, like below:

```suggestion
* **New Provider Config**: `organization` (or the `TFE_ORGANIZATION` environment variable) defines a default organization for all resources, making all resource-specific organization arguments optional. @brandonc ([#762](https://github.com/hashicorp/terraform-provider-tfe/pull/762))
```

What are your thoughts?